pages/linux/x11vnc.md

```diff
@@ -0,0 +1,15 @@
+# x11vnc
+
+> A VNC server for real X Displays.
```

Comment:
It may not be clear what "real" means here. Can you think of a concise way to explain the term?